### PR TITLE
[16.01] Fix create/manage db scripts to respect GALAXY_VIRTUAL_ENV.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

**Comment by bgruening** [comment]
@jmchilton is this related? https://github.com/galaxyproject/galaxy/pull/1816
Maybe there are even more scripts that need to be updated?

File: create_db.sh [hunk 1]
```diff
@@ -1,9 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (nsoranzo) [inline create_db.sh]: I can't see a reason for this change (same below).

Comment (jmchilton) [inline create_db.sh]: Common startup assumes bash, I guess I did this for consistency with that script. I can revert the change if you wish.

Comment (nsoranzo) [inline create_db.sh]: Actually Galaxy starts fine for me on Ubuntu (which uses dash) when changing `s/bash/sh/` in `scripts/common_startup.sh` .

Comment (bgruening) [inline create_db.sh]: This is because /bin/sh is symlinked to /bin/dash right?
I think we should use /bin/bash as this is needed for example by conda.

Comment (jmchilton) [inline create_db.sh]: I'll revert this PR - I guess this isn't the place to decide what common_startup.sh should be.

**Comment by jmchilton** [comment]
Removed the references to bash.

**Comment by martenson** [comment]
:+1:

**Pull request merged**